cmd/internals/schema-registry-light-client.go

```diff
@@ -282,10 +282,32 @@ func (src *SchemaRegistryClient) GetSchema(subject string, version int64, delete
 
 	err = json.Unmarshal(body, &schemaResponse)
 	checkDontFail(err)
-
+	
+	//update or append doc field with SR subject name
+	
+	for _, arg := range os.Args {
+		if arg == "-"+"addDocField" {
+			subjectName := schemaResponse.Subject
+
+			// Adding the new field to the schemas after importing and before exporting
+			var unmarshalledSchema map[string]interface{}
+			json.Unmarshal([]byte(schemaResponse.Schema), &unmarshalledSchema)
+			key, exists := unmarshalledSchema["doc"]
+
+			if exists {
+				value := key.(string)
+				unmarshalledSchema["doc"] = subjectName + " - " + value
```

Comment:
since we are aiming to make schemas unique, we'd need to update the doc field also if one already exists to ensure every schema with a different ID is unique. This change is to workaround the issue where Apicurio allows schemas with identical configuration be registered with different IDs